pkg/helpers/image/test/util.go

```diff
@@ -37,10 +37,13 @@ const (
 )
 
 // ImageList turns the given images into ImageList.
-func ImageList(images ...imagev1.Image) imagev1.ImageList {
-	return imagev1.ImageList{
-		Items: images,
+func ImageList(images ...imagev1.Image) map[string]*imagev1.Image {
```

Comment:
Rename this function? It does not "turn the given images into ImageList" (anylonger) :-D

Reply:
done